tools/integration_tests/tokenserver/mock_fxa_server.py

```diff
@@ -0,0 +1,72 @@
+from wsgiref.simple_server import make_server as _make_server
+from pyramid.config import Configurator
+from pyramid.response import Response
+from pyramid.view import view_config
+import json
+import os
+
+
+@view_config(route_name='mock_oauth_verify', renderer='json')
+def _mock_oauth_verify(request):
```

Comment:
These mock verify endpoints work the same way as [this mock server](https://github.com/mozilla-services/tokenserver/blob/master/loadtest/mock-oauth-cfn.yml) from the old Tokenserver's load tests. Instead of sending an opaque JWT token, the client sending the request to Tokenserver encodes the response it desires from FxA in the `Authorization` header (e.g. `Authorization: { "body": { .. }, "status": 200 }

Note that this server is stubbing out FxA, so the flow is:
User --> Tokenserver --> FxA

Tokenserver is oblivious to the fact that the client is sending it bogus tokens since it just passes it along to the verification server.

Reply:
That exception is from a test that sends it improperly-formatted JSON -- an exception is thrown by `json.loads` and a 5XX error is returned. Do you think we should catch that exception and return a response explicitly?